src/index.ts

```diff
@@ -3,12 +3,39 @@ import { connectToParent } from "penpal";
 interface AssistedCallStartParams {
   phoneNumber: string;
 }
+
+interface ContactModifyParams {
+  first_name: string | null;
```

Comment:
The parameters seems mandatory. If I would just want to change e.g. name I would need to provide all the other properties as well.